### ISSUE TITLE
UefiPayloadPkg: payload error (Bugzilla Bug 3585)

### ISSUE THREAD

**Issue by tianocore-issues** [tianocore/edk2#8952]
This issue was created automatically with [bugzilla2github](https://github.com/makubacki/bugzilla2github/tree/bz2gh_tianocore)
# Bugzilla Bug 3585
Date: 2021-08-24T11:52:18+00:00
From: Sumo &lt;<kingsumos>&gt;
To:   @LiuZhiguang001
CC:   @lgao4

Last updated: 2021-09-08T22:04:18+00:00

**Comment by tianocore-issues** [comment]
## Comment 17022
Date: 2021-08-24 11:52:18 +0000
From: Sumo &lt;<kingsumos>&gt;

- Industry Specification: ---
- Release Observed: EDK II Master
- Releases to Fix: EDK II Master
- Target OS: ---
- Bugzilla Assignee(s): @LiuZhiguang001

The following patch broke the payload loading when using Coreboot.

commit 4bac086e8e007c7143e33f87bb96238326d1d6ba
Author: Zhiguang Liu &lt;zhiguang.liu@intel.com&gt;
Date:   Wed Jul 14 14:24:45 2021 +0800

    UefiPayloadPkg: Add FV Guid for DXEFV and PLDFV

    Signed-off-by: Zhiguang Liu &lt;zhiguang.liu@intel.com&gt;
    Reviewed-by: Ray Ni &lt;ray.ni@intel.com&gt;
    Reviewed-by: Guo Dong &lt;guo.dong@intel.com&gt;

Basically the coreboot cbfstool reports the following error when creating the CBFS / flash image:
&quot;Not a usable UEFI firmware volume&quot;

Trying to boot coreboot results in an exception and the following error message:
&quot;Payload not loaded&quot;

Link to discussion:
https://edk2.groups.io/g/devel/message/79729

**Comment by tianocore-issues** [comment]
## Comment 17036
Date: 2021-08-24 21:43:25 +0000
From: @lgao4

Zhiguang, please check.

**Comment by tianocore-issues** [comment]
## Comment 17155
Date: 2021-09-02 14:30:55 +0000
From: Sumo &lt;<kingsumos>&gt;

Support must be added in coreboot, se also:
https://review.coreboot.org/c/coreboot/+/57296

**Comment by tianocore-issues** [comment]
## Comment 17228
Date: 2021-09-08 22:04:18 +0000
From: @LiuZhiguang001


The issue happens because coreboot didn&apos;t support extra FV header before. we already add the support in coreboot side at https://review.coreboot.org/c/coreboot/+/57296. I will close this bugzilla as fixed. If it is wrong, please inform me.
